### ISSUE TITLE
Microservices Config File Missing

### ISSUE THREAD

**Issue by twinkie0101** [imagegenius/docker-immich#9]
Trying to setup Immich via Unraid using community applications.

Installed Immich, Redis and PostgesSQL 14 but when starting Immich getting this error:

`./run: line 6: /config/log/immich/microservices.log: No such file or directory`

Here's additional information from the log if this helps:

```
-------------------------------------
|    _               _              |
|   | |             | |             |
|   | |__  _   _  __| | __ _ ____   |
|   | '_ \| | | |/ _` |/ _` |_  /   |
|   | | | | |_| | (_| | (_| |/ /    |
|   |_| |_|\__, |\__,_|\__,_/___|   |
|           __/ |                   |
|          |___/                    |
-------------------------------------

To support the app dev(s) visit:
Immich: https://www.buymeacoffee.com/altran1502
-------------------------------------
User/Group ID
-------------------------------------
User UID: 911
User GID: 911
-------------------------------------
[custom-init] No custom files found, skipping...
./run: line 6: /config/log/immich/microservices.log: No such file or directory
nginx: [warn] duplicate MIME type "text/html" in /defaults/nginx.conf:37
Listening on 0.0.0.0:3000
2023-01-21 23:20:58.152302: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [NestFactory] Starting Nest application...
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [ImageClassifier] Running Node TensorFlow Version : 3.19.0
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [ObjectDetection] Running Node TensorFlow Version : 3.19.0
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] AppModule dependencies initialized +2ms
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ImageClassifierModule dependencies initialized +1ms
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ObjectDetectionModule dependencies initialized +0ms
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [RoutesResolver] ImageClassifierController {/image-classifier}: +3ms
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [RouterExplorer] Mapped {/image-classifier/tag-image, POST} route +2ms
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [RoutesResolver] ObjectDetectionController {/object-detection}: +0ms
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [RouterExplorer] Mapped {/object-detection/detect-object, POST} route +0ms
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [NestApplication] Nest application successfully started +2ms
[Nest] 140  - 01/21/2023, 11:20:58 PM     LOG [IMMICH MICROSERVICES] Running Immich Machine Learning in PRODUCTION environment
[Nest] 142  - 01/21/2023, 11:20:58 PM    WARN The current JWT_SECRET key is insecure. It should be at least 128 bits long!
[Nest] 142  - 01/21/2023, 11:20:58 PM    WARN Here is a new, securely generated key that you can use instead: 2qiSKo1v9N6+ObAh9g+i2TAk04yzzKUzI1fO/tVMv2o=
./run: line 6: /config/log/immich/microservices.log: No such file or directory
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [NestFactory] Starting Nest application... +119ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +45ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] DownloadModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] BullModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] JwtModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ConfigHostModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] DiscoveryModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ConfigModule dependencies initialized +10ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ScheduleModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] BullModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] BullModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] BullModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] BullModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] BullModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] BullModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] BackgroundTaskModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmCoreModule dependencies initialized +141ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] InfraModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ScheduleTasksModule dependencies initialized +2ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] DomainModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] AppModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ImmichJwtModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] CommunicationModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] DeviceInfoModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ServerInfoModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] TagModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] AlbumModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] SystemConfigModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ShareModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] ImmichConfigModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] StorageModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] OAuthModule dependencies initialized +3ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] JobModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] AuthModule dependencies initialized +0ms
[Nest] 142  - 01/21/2023, 11:20:58 PM     LOG [InstanceLoader] AssetModule dependencies initialized +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] AppController {/}: +91ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/refresh-config, POST} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] APIKeyController {/api-key}: +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/api-key, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/api-key, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/api-key/:id, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/api-key/:id, PUT} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/api-key/:id, DELETE} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] UserController {/user}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user/info/:userId, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user/me, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user/count, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user/:userId, DELETE} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user/:userId/restore, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user, PUT} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user/profile-image, POST} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/user/profile-image/:userId, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] AssetController {/asset}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/upload, POST} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/download/:assetId, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/download-files, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/download-library, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/file/:assetId, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/thumbnail/:assetId, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/curated-objects, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/curated-locations, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/search-terms, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/search, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/count-by-time-bucket, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/count-by-user-id, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/time-bucket, POST} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/:deviceId, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/assetById/:assetId, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/:assetId, PUT} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset, DELETE} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/check, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/exist, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/shared-link, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/asset/shared-link, PATCH} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] ShareController {/share}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/share, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/share/me, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/share/:id, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/share/:id, DELETE} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/share/:id, PATCH} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] TagController {/tag}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/tag, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/tag, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/tag/:id, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/tag/:id, PATCH} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/tag/:id, DELETE} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] AlbumController {/album}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/count-by-user-id, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/:albumId/users, PUT} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/:albumId/assets, PUT} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/:albumId, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/:albumId/assets, DELETE} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/:albumId, DELETE} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/:albumId/user/:userId, DELETE} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/:albumId, PATCH} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/:albumId/download, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/album/create-shared-link, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] AuthController {/auth}: +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/auth/login, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/auth/admin-sign-up, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/auth/validateToken, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/auth/change-password, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/auth/logout, POST} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] OAuthController {/oauth}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/oauth/mobile-redirect, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/oauth/config, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/oauth/callback, POST} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/oauth/link, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/oauth/unlink, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] DeviceInfoController {/device-info}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/device-info, POST} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/device-info, PATCH} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/device-info, PUT} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] ServerInfoController {/server-info}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/server-info, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/server-info/ping, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/server-info/version, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/server-info/stats, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] JobController {/jobs}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/jobs, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/jobs/:jobId, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/jobs/:jobId, PUT} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RoutesResolver] SystemConfigController {/system-config}: +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/system-config, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/system-config/defaults, GET} route +1ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/system-config, PUT} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [RouterExplorer] Mapped {/system-config/storage-template-options, GET} route +0ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [NestApplication] Nest application successfully started +25ms
[Nest] 142  - 01/21/2023, 11:20:59 PM     LOG [ImmichServer] Running Immich Server in PRODUCTION environment - version 1.42.0
./run: line 6: /config/log/immich/microservices.log: No such file or directory
```
The no file or directory persists,  Not sure what else to provide, but happy to help where I can.

**Comment by LlamaSauce** [comment]
This appears to impact jobs running, I uploaded photos but the generate thumbnails was not working.

I created a folder under the log dir named immich and my jobs instantly kicked in.

**Comment by ptichalouf** [comment]
it's the same on unraid

**Comment by twinkie0101** [comment]
I have Immich installed on unraid, but not sure how to create the directory as no appdata folder is created when installing the Immich. Would appreciate any help you can provide.

**Comment by ptichalouf** [comment]
> I have Immich installed on unraid, but not sure how to create the directory as no appdata folder is created when installing the Immich. Would appreciate any help you can provide.

you don't need to do anything just point the directery to your photo when you install the container and that all. just the latest release has a bug

**Comment by LlamaSauce** [comment]
> I have Immich installed on unraid, but not sure how to create the directory as no appdata folder is created when installing the Immich. Would appreciate any help you can provide.

Click on the Immich icon under Docker in Unraid and select **Console**.

When the console window opens type : **mkdir config/log/immich** and hit enter on your keyboard.

**Comment by ptichalouf** [comment]
> > I have Immich installed on unraid, but not sure how to create the directory as no appdata folder is created when installing the Immich. Would appreciate any help you can provide.
> 
> Click on the Immich icon under Docker in Unraid and select **Console**.
> 
> When the console window opens type : **mkdir config/log/immich** and hit enter on your keyboard.

Perfect, thanks for your time

**Comment by hydazz** [comment]
Sorry about this, it is fixed in https://github.com/imagegenius/docker-immich/commit/e6961fec2a59e6e3a1433e245154da1b41773b1c. once https://ci.imagegenius.io/blue/organizations/jenkins/Docker-Pipeline-Builders%2Fdocker-immich/detail/main/67/pipeline finishes you can update the image and you can remove the `config/log/immich` folder. LMK how it goes after updating

**Comment by ptichalouf** [comment]
np answer here when it's out if u want a test really quick

**Comment by ptichalouf** [comment]
> Sorry about this, it is fixed in [e6961fe](https://github.com/imagegenius/docker-immich/commit/e6961fec2a59e6e3a1433e245154da1b41773b1c). once https://ci.imagegenius.io/blue/organizations/jenkins/Docker-Pipeline-Builders%2Fdocker-immich/detail/main/67/pipeline finishes you can update the image and you can remove the `config/log/immich` folder. LMK how it goes after updating

np answer here when it's out if u want a test really quick

**Comment by hydazz** [comment]
fixed in https://github.com/imagegenius/docker-immich/commit/e6961fec2a59e6e3a1433e245154da1b41773b1c
